packages/esm-service-queues-app/src/queue-table/cells/queue-table-status-cell.component.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { type QueueTableColumn, type QueueTableCellComponentProps } from '../../types';
+import QueueStatus from '../../queue-entry-table-components/queue-status.component';
 
 export const QueueTableStatusCell = ({ queueEntry }: QueueTableCellComponentProps) => {
-  return <>{queueEntry.status.display}</>;
+  return <QueueStatus status={queueEntry.status} />; // Do not pass queue into status, as we do not want to render it
```

Comment:
not sure what the comment meant...

Reply:
Might be a hacky way to implement it, but if you pass both status and queue into the tag, then it will render the status as `status.display - queue.display`.  If you only pass the status in, then it will render it as `status.display` only.  So in views like the active visits table in which you may be viewing multiple queues at a location in one table, this allows the pre-existing behavior to continue to work, where both are shown.  But in the queue-table-by-status component, which only ever shows entries from a single queue, it allows that pre-existing behavior to work by excluding the queue name from the status display.